### ISSUE TITLE
interface bug in Bun

### ISSUE THREAD

**Issue by zolomatok** [DanielXMoore/Civet#299]
The following **TS** code runs perfectly fine with **Bun**:
```
interface Something {
    name: string
}

let a: Something = { name: "" }
```

Similarly, this **Civet** code has no problem being ran with `civet test.civet`:
```
interface Something
    name: string

let a: Something = 
    name: ""
```

But running `bun main.ts` where **main.ts** is:
```
// @ts-nocheck
import "@danielx/civet/bun-civet"
import * as test from './test.civet'
test
```
and **test.civet** is the **Civet** code above, I get the following error:

```
error: Expected ";" but found ":"

let a: Something =  {
     ^
/Users/zolo/project/program/test.civet:4:6 43
```

The reason I think this is a Civet issue is that the TS equivalent runs without issues in Bun.

Only `interface` seems to cause this problem, I haven't run into any other issues yet.

**Comment by zolomatok** [comment]
My Civet version is **0.5.56**

**Comment by STRd6** [comment]
Fixed in 0.5.59
